display_list/testing/dl_test_surface_provider.cc

```diff
@@ -10,50 +10,29 @@
 #include "third_party/skia/include/core/SkSurface.h"
 #include "third_party/skia/include/encode/SkPngEncoder.h"
 
-#ifdef ENABLE_SOFTWARE_BENCHMARKS
-#include "flutter/display_list/testing/dl_test_surface_software.h"
-#endif
-#ifdef ENABLE_OPENGL_BENCHMARKS
-#include "flutter/display_list/testing/dl_test_surface_gl.h"
-#endif
-#ifdef ENABLE_METAL_BENCHMARKS
-#include "flutter/display_list/testing/dl_test_surface_metal.h"
-#endif
-
-namespace flutter {
-namespace testing {
+namespace flutter::testing {
 
 std::string DlSurfaceProvider::BackendName(BackendType type) {
   switch (type) {
-    case kMetalBackend:
-      return "Metal";
-    case kOpenGlBackend:
-      return "OpenGL";
     case kSoftwareBackend:
       return "Software";
+    case kOpenGlBackend:
+      return "OpenGL";
+    case kMetalBackend:
+      return "Metal";
```

Comment:
Reordered for consistency with elsewhere in the file, and with the declaration order in the enum.